documentation/how-to/pg-service.en.md

```diff
@@ -1,96 +1,151 @@
 
-We first need to set up a configuration file.
-There are many options to organize this, [read more in the PostgreSQL documentation](https://www.postgresql.org/docs/current/libpq-pgservice.html)<!-- markdown-link-check-disable-line --> or follow the description below.
+    1. In your Browser right-click on the *PostGIS Elephant* > *New Connection*
+    2. Instead of entering all your the database information, just add your service name and "Test the connection".
+    If successful, click "Ok"
 
-1. **Create a Configuration File**:
+## Connection to PostGIS in QField
 
-   - On Windows:
-     Create a file named `pg_service.conf` and store it in a convenient location.
+If you have connected to PostGIS using the PG Service file and not directly, it is also necessary to add the information either on your mobile device directly or within QFieldCloud as a "Secret".
 
-   - On Linux/MacOS/Unix:
-     Create a file named `.pg_service.conf` in your home folder (`~`).
+### Configuration on Mobile Device
+
+If you copy your projects via cable to your device, you will also have to copy the information of the database to the right right directory on your device.
+Generally, the QField directory for Android can be found under `/Android/data/ch.opengis.qfield/files/QField.`
+!!! Note
+
+   Due to the restrictions of Android, you will only be able to access the directory when being connected via cable to the computer.
+   The only other possible option you have is to set your device to root (not recommended)
+
+### Configuration on QFieldCloud
+
+QFieldCloud uses the concept of "Secrets".
+Secrets need to be configured in QFieldCloud as an additional setting.
+They store the information of your database in an encrypted way and are hence, the recommended way to give access to your database.
+Read more on [Secrets](../reference/qfieldcloud/secrets.md) in the QFieldCloud documentation.
+There you will also find more information on configuring user-defined secrets.
+
+!!! Workflow
+
+    1. From your landing page on QFieldCloud direct to your project
+    2. Click on the *Secrets* section.
+    3. Click on "Add New Secret" and choose the second option "Add a pg_service.conf configuration"
+    4. Now you have two options:
+          - Enter your database information again (host, port, SSL mode, password, username) OR
+          - Click on advanced editor and simply copy the lines from your previously established servicefile.
+          **Note:** You can also just use the clipboard function inside the PG service parser plugin
 
-2. **Define Connection Parameters**:
+    !![Add the secret](../assets/images/service_config_file_004.png,350px)
 
-   Within the file, specify connection parameters for your PostgreSQL database using the following format:
+### Advanced Configuration Details
 
-   ```ini
-   [SERVICE_NAME]
-   host=your_host_or_ip
-   port=your_port
-   dbname=your_database_name
-   user=your_username
-   password=your_password
-   ```
+#### Storage Paths
```

Comment:
Since we're recommending that users use the plugin, remove this section.

The plugin creates the proper file in the proper directory, regardless of the operating system.